project/AppDependencies.scala

```diff
@@ -2,13 +2,23 @@ import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  val bootStrapPlayVersion = "5.3.0"
+  private lazy val bootStrapPlayVersion = "6.4.0"
+  private lazy val silencerVersion      = "1.7.9"
 
-  val compile: Seq[ModuleID] = Seq(
-    ws,
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % bootStrapPlayVersion)
+  private lazy val silencerDependencies: Seq[ModuleID] = Seq(
+    compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
+    "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
+  )
+
+  private lazy val compile: Seq[ModuleID] =
+    Seq(ws, "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % bootStrapPlayVersion)
+
+  private lazy val test: Seq[ModuleID]    = Seq(
+    "org.pegdown"          % "pegdown"      % "1.6.0",
```

Comment:
Can this be removed please since it is not needed? See [here](https://github.com/sirthias/pegdown#-deprecation-note-) for more info. Thanks

Reply:
I have seen this one is used to generate some reports I know is deprecated but as far as I can see other repos are including it for the reports, I can try to remove and see if report is still generated or not but not sure